packages/back-end/package.json

```diff
@@ -34,7 +34,7 @@
     "@dqbd/tiktoken": "^1.0.7",
     "@google-cloud/bigquery": "5",
     "@google-cloud/storage": "^5.20.5",
-    "@growthbook/growthbook": "^1.0.1",
+    "@growthbook/growthbook": "1.0.1",
```

Comment:
Prevented all the packages from upgrading to 1.0.2 until we're ready. I'll swap these deps back to use `^` when I add the supporting code and bump the version req